kubernetes/samples/scripts/domain-lifecycle/helper.sh

```diff
@@ -71,6 +71,25 @@ function getEffectivePolicy {
   eval $__currentPolicy=${currentPolicy}
 }
 
+#
+# Function to get effective start policy of admin server
+# $1 - Domain resource in json format
+# $2 - Return value containing effective server start policy
+#      Legal retrun values are "NEVER" or "IF_NEEDED" or "ALWAYS".
+#
+function getEffectiveAdminPolicy {
+  local domainJson=$1
+  local __effectivePolicy=$2
+  local __adminStartPolicy=""
```

Comment:
Can __domainStartPolicy be declared local? Or will that not work?

Reply:
Done.